src/mn_setup.c

```diff
@@ -2054,6 +2055,8 @@ static void SetMidiPlayer(void)
     S_RestartMusic();
 }
 
+static const char *equalizer_preset_strings[] = {"Off", "Bass", "Bass+Treble", "Mid"};
```

Comment:
When the preset names are finalized, check if the names are cut off when using a wad that has a wide font, like Eviternity II:

![woof0000](https://github.com/fabiangreffrath/woof/assets/56656010/86fddc18-d32b-41e3-a97f-0e8f2b349ce6)